client/components/data/query-jetpack-modules/test/query-jetpack-modules.jsx

```diff
@@ -49,6 +51,9 @@ function getStore() {
 
 describe( 'fetchModuleList', () => {
 	test( "Ensure we're NOT calling fetchModuleList for simple sites", async () => {
+		isWpcomAtomic = false;
+		isJetpack = false;
```

Comment:
If we always declare these in the beginning of the test, we might as well remove the `let` call and move them here as `const`s?

Reply:
I'm not sure if I understood your suggestion since we're using these variables inside `mockStore`, and if we declare them constants, we can't change them on other tests.

Reply:
I see, that makes sense. Let's leave them as they are then 👍